include/soci/row.h

```diff
@@ -51,31 +65,25 @@ class SOCI_DECL row
     indicator get_indicator(std::size_t pos) const;
     indicator get_indicator(std::string const& name) const;
 
-    template <typename T>
-    inline void add_holder(T* t, indicator* ind)
-    {
-        holders_.push_back(new details::type_holder<T>(t));
-        indicators_.push_back(ind);
-    }
-
     column_properties const& get_properties(std::size_t pos) const;
     column_properties const& get_properties(std::string const& name) const;
 
     template <typename T>
     T get(std::size_t pos) const
     {
         typedef typename type_conversion<T>::base_type base_type;
-        base_type const& baseVal = holders_.at(pos)->get<base_type>();
+        base_type baseVal;
+        get_(pos, baseVal);
```

Comment:
Could we please rename this `get_` to `get_at()` or `do_get()` or something else, but to without `_` at the end of the name? As it is, this is a bit confusing, as this naming convention is used for the member variables in SOCI, but this one is a function, not a variable.